bounties/npm/selenium-wrapper/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Resources Downloaded over Insecure Protocol",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "selenium-wrapper",
+		"URL": "https://www.npmjs.com/package/selenium-wrapper",
+		"Downloads": "1486"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-494",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "H",
+		"PR": "L",
+		"UI": "R",
+		"S": "U",
+		"C": "H",
+		"I": "H",
+		"A": "H",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.1"
+	},
+	"CVEs": [
+		"CVE-2016-10628"
+	],
+	"Repository": {
+		"URL": "https://github.com/arian/selenium-wrapper",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [
+		"https://github.com/arian/selenium-wrapper/blob/0b6d75021f54c1dd2a1be8c227e37b8c237723d0/install.js#L11"
```

Comment:
Just change the link to the `master` branch: https://github.com/arian/selenium-wrapper/blob/master/install.js#L11

Reply:
Sorted, cheers 😁